### DISCUSSION TITLE
Can I Reset gh-pages?

### DISCUSSION THREAD

**Discussion by mitchelldmiller** [community/community#22954] [Other Feature Feedback, Questions, & Ideas]
<p>Hello:</p>
<p>I forgot to update my repository’s <a href="https://mitchelldmiller.github.io/quick-mail-wp-plugin/" rel="nofollow noopener">Github page</a> for a few years.</p>
<p>I think I turned on autorebase, and did not realize I needed a separate</p>
<pre><code>git rebase master
</code></pre>
<p>to generate page. Scary that I never checked the page.</p>
<p>Changing theme did not help.</p>
<p>How can I force Github to generate a new page from current README.md ?</p>
<p>There are many conflicts.</p>
<p>Thank you,</p>
<p>Mitchell</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/mitchelldmiller">@mitchelldmiller</a>,</p>
<p>Thank you for being here! Looks like your page is rending from the index.md file at this moment. A change there should trigger a new build. I hope this helps!</p>

**Comment by mitchelldmiller** [comment]
<p>Hello AndreaG:</p>
<p>Thank you for reply.</p>
<blockquote>
<p>Looks like your page is rending from the index.md file at this moment. A change there should trigger a new build. I hope this helps!</p>
</blockquote>
<p>I checked out gh-pages and copied README.md and index.md. Pushed gh-pages.</p>
<p>However, page did not change until I edited Settings -&gt; Github Pages -&gt; Source to gh-pages.</p>
<p>Best wishes,</p>
<p>Mitchell</p>
